python/__init__.py

```diff
@@ -72,6 +72,27 @@ def __hash__(self):
 except ImportError:
     pass
 
+# Define a `with edit(layer)` statement
```

Comment:
Yeah those can move too.  I think we did that before I created the core `__init__` files.